configure.ac

```diff
@@ -1,13 +1,14 @@
 AC_PREREQ([2.61])
 dnl Don't forget when updating version number to do GP_CHECK_GEANY also
 AC_INIT([geany-plugins], [2.1])
-AM_INIT_AUTOMAKE([1.8 foreign dist-bzip2 tar-ustar subdir-objects])
-
-m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
```

Comment:
@eht16 can you tell whether moving this was needed or not in the end?

Reply:
I'm testing without, we'll see

Reply:
(it's actually required to have `AM_INIT_AUTOMAKE` *after* `AC_CONFIG_AUX_DIR` on all platforms… so if I had tested before asking I'd have known).